pkg/mimeutils/type.go

```diff
@@ -0,0 +1,213 @@
+// Copyright 2010 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
```

Comment:
I guess we should adjust then the .reuse file to apply the correct license to this file!?!